src/main/java/teammates/common/util/Const.java

```diff
@@ -428,6 +428,23 @@ public static final String getjQueryUiFilePath(boolean isDevServer) {
     
     public static class FeedbackQuestion {
 
+        public static final Map<String, String>
+                COMMON_VISIBILITY_OPTIONS;
+
+        static {
+            Map<String, String> visibilityOptionInit =
+                    new LinkedHashMap<String, String>();
+
+            visibilityOptionInit.put("ANONYMOUS_TO_RECIPIENT_AND_INSTRUCTORS",
```

Comment:
I kept this as a map for consistency with common feedback path options (which is slightly too complicated for enum)